.github/workflows/build-and-test.yml

```diff
@@ -54,15 +54,17 @@ jobs:
     - name: Generate OpenAPI spec
       run: ./gradlew --stacktrace :core:generateOpenApiSpec
 
+    - name: Install pnpm
```

Comment:
Interesting, this is doing the opposite to what I was mentioning in the previous commit. Maybe better link to https://github.com/actions/setup-node?tab=readme-ov-file#usage, quoting "Package manager should be pre-installed" in the commit message?